<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Layer</class>
 <widget class="QMainWindow" name="Layer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>538</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Layer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #444444;
    color: #FFFFFF;</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="autoExpandDelay">
     <number>-1</number>
    </property>
    <property name="indentation">
     <number>0</number>
    </property>
    <property name="rootIsDecorated">
     <bool>true</bool>
    </property>
    <property name="uniformRowHeights">
     <bool>false</bool>
    </property>
    <property name="itemsExpandable">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="animated">
     <bool>false</bool>
    </property>
    <property name="allColumnsShowFocus">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="headerHidden">
     <bool>false</bool>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="headerCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="headerMinimumSectionSize">
     <number>21</number>
    </attribute>
    <attribute name="headerDefaultSectionSize">
     <number>70</number>
    </attribute>
    <attribute name="headerHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="headerShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>图标</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>冻结</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>显示为外框</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
     <property name="icon">
      <iconset resource="main.qrc">
       <normaloff>:/Resource/Layer/ble_on_32.png</normaloff>:/Resource/Layer/ble_on_32.png</iconset>
     </property>
     <property name="text">
      <string>0(默认)</string>
     </property>
     <property name="icon">
      <iconset>
       <selectedon>:/Resource/Layer/Layers_48.png</selectedon>
      </iconset>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string>层001</string>
     </property>
     <property name="icon">
      <iconset>
       <selectedon>:/Resource/Layer/Layers_48.png</selectedon>
      </iconset>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd"/>
  </widget>
  <widget class="QMenuBar" name="menuCustomMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>356</width>
     <height>20</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuselect">
    <property name="title">
     <string>select</string>
    </property>
   </widget>
   <addaction name="menuselect"/>
  </widget>
  <action name="actionadd">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/Layer/CreateNewLayer_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>add</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
